docs/machineregistration-reference.md

```diff
@@ -75,9 +91,10 @@ This refers to the name that will be set to the node and the kubernetes resource
 `String` type.
 
 !!! info
-    machineName is interpolated with [SMBIOS](https://www.dmtf.org/standards/smbios) data which allows you to store hardware
-    information.
-    See our [SMBIOS docs](smbios.md) for more information
+    when `elemental:registration:no-smbios` is set to `false` (default) machineName is interpolated with [SMBIOS](https://www.dmtf.org/standards/smbios) data which allows you to store hardware information.
```

Comment:
can you add this line below also ot have consistent notes about smbios (line 116)?

Reply:
yep, good catch, I missed that!

Reply:
done